.github/workflows/python-test.yml

```diff
@@ -54,3 +54,39 @@ jobs:
       - name: Run pytest
         working-directory: python
         run: poetry run pytest tests -m "not slow"
+
+      - name: Code Coverage Report
+
+        uses: irongut/CodeCoverageSummary@v1.3.0
+
+        with:
+
+          filename: coverage/**/coverage.cobertura.xml
+
+          badge: true
+
+          fail_below_min: true
+
+          format: markdown
+
+          hide_branch_rate: false
+
+          hide_complexity: true
+
+          indicators: true
+
+          output: both
+
+          thresholds: '60 80'
+
+    - name: Add Coverage PR Comment
```

Comment:
Thank you for your contribution!
The indentation of this line breaks the parsing of the file. Could you fix it?

Reply:
I hope indentation fixed